tasks/sast-java-sec-check.yaml

```diff
@@ -62,6 +62,7 @@ spec:
 
         ERR_MSG="$(cat stderr.txt)"
```

Comment:
Thanks for making the fix, I tried and it looks changing this line to `ERR_MSG="$(echo -n $(cat stderr.txt))"` can remove the blank lines as well.

Reply:
Ack, updated